hera_pspec/uvpspec.py

```diff
@@ -26,7 +26,14 @@ def __init__(self):
         by hera_pspec.
         """
         # Summary attributes
-        self._Ntimes = PSpecParam("Ntimes", description="Number of unique times.", expected_type=int)
+        # Note that in the past (pre-v0.5) Ntimes was the number of unique times in 
+        # either of the underlying visibilitydatasets (i.e. time_1_array and time_2_array),
+        # however throughout the code it was often assumed to mean the number of unique
+        # time-pairs (i.e. how many distinct average times there are in the data), which
+        # is really Ntpairs. In fact, there's never a situation in which you would need
+        # to know the number of unique underlying times in the context of power spectra,
+        # so now we make the two attributes identical.
```

Comment:
Should we just remove one? I guess `uvp.Ntimes`? Or we could keep this change for a future release

Reply:
I definitely considered removing it, but I think maybe that would break more code than we intend to -- it might have quite significant downstream effects. I'm already scared about re-interpreting the meaning. I think I got all the usages inside the pspec repo sorted out, but I don't know about usages outside the repo, in pipeline code...